readme.md

```diff
@@ -48,7 +48,7 @@ Milliseconds to humanize.
 
 #### options
 
-##### secDecimalDigits
+##### sec-decimal-digits
```

Comment:
Kinda unrelated, but do you have any suggestion on a shorter option name for this than `secDecimalDigits`?